### ISSUE TITLE
use on other platforms

### ISSUE THREAD

**Issue by csadmin22** [opncell/opncell#16]
is this able to be used on platforms like Mikrotik router OS

**Comment by rhamblet** [comment]
No, This is compiled specifically for FreeBSD 14.x Retargeting to RouterOS would be quite a bit of effort.
